### PR TITLE
feat: LSDV-4583-3: Image list navigation with hotkeys

### PR DESCRIPTION
**Pull request by nick-skriabin**
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [x] Tests for the changes have been added/updated (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [x] Frontend

### Describe the reason for change
LSDV-4583
LSDV-4583-3

#### What is the new behavior?
This PR enables navigation between pages using
- `Ctrl+[` to go prev
- `Ctrl+]` to go next

#### What is the current behavior?
Previously users could only navigate between images using mouse/trackpad and pagination interface

#### What libraries were added/updated?
None

#### Does this change affect performance?
No

#### Does this change affect security?
No

#### What alternative approaches were there?
None

#### What feature flags were used to cover this change?
`feat_front_lsdv_4583_multi_image_segmentation_short`

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [x] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [x] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
Labeling Interface, Image Segmentation

### CODE REVIEW

File: src/common/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -135,15 +150,26 @@ export const Pagination: FC<PaginationProps> = forwardRef<any, PaginationProps>(
 });
 
 const NavigationButton: FC<{
-  onClick: (e: React.MouseEvent<HTMLDivElement>) => void,
+  onClick: () => void,
   mod: string[],
   disabled?: boolean,
+  hotkey?: string,
 }> = (props) => {
```
Comment (yyassi-heartex) [inline src/common/Pagination/Pagination.tsx:157]: can we change props to `{ onClick, mod, disabled, hotkey }` and reference the prop values accordingly? just to be more consistent with the rest of our code

**Review by yyassi-heartex — APPROVED** [review]
looking good, just a minor optional tweak which seems to have been there from before but should be updated to be more consistent with the rest of our code

File: src/core/settings/keymap.json [hunk 2]
```diff
@@ -162,5 +162,13 @@
   "repeater:previous-page": {
     "key": "alt+left",
     "description": "Previous Page"
+  },
+  "image:prev": {
+    "key": "ctrl+[",
+    "description": "Next Image"
```
Comment (bmartel) [inline src/core/settings/keymap.json:168]:
```suggestion
    "description": "Previous Image"
```

**Review by bmartel — CHANGES_REQUESTED** [review]
Just a small typo in the key previous description, this looks awesome 🔥

**Review by bmartel — APPROVED** [review]

**Pull request closed without merging**